docs/src/main/asciidoc/rest-json-guide.adoc

```diff
@@ -377,6 +377,64 @@ Let's do that and follow the same steps as before:
 
 This time, you can see our list of legumes.
 
+== HTTP filters and interceptors
+
+Both HTTP request and response can be intercepted by providing `ContainerRequestFilter` or `ContainerResponseFilter`
+implementations respectively. These filters are suitable for processing the metadata associated with a message: HTTP
+headers, query parameters, media type, and other metadata. They also have the capability to abort a REST invocation
```

Comment:
The word REST is not great here. Maybe something like: 

```
They also have the capability to abort the request processing, for instance when the user does not have the permissions to access the endpoint.
```

WDYT?

Reply:
Yes, what you proposed is much better. Will change this sentence.

Reply:
Done.